include/lsst/afw/detection/Footprint.h

```diff
@@ -70,13 +70,30 @@ class Footprint : public lsst::daf::base::Citizen,
      * that to construct the Footprint.
      */
 
+    /** @brief Constructor for Footprint object
+     *
+     * @param inputSpans Shared pointer to a SpanSet defining the pixels included in
+                         the Footprint.
+     * @param region Bounding box of the MaskedImage in which the Footprint was created,
```

Comment:
oh good point, I just pulled over from old footprints. I maybe should just change that to lowercase image to just give idea of generic image.